packages/db/src/migrations/20250425134823_granules_add_producer_granule_id.ts

```diff
@@ -13,9 +13,9 @@ export const up = async (knex: Knex): Promise<void> => {
     await knex.schema.table('granules', (table) => {
       table.text('producer_granule_id').notNullable().alter();
     });
-  }
 
-  await knex.raw('CREATE INDEX CONCURRENTLY IF NOT EXISTS granules_producer_granule_id_index ON granules(producer_granule_id)');
+    await knex.raw('CREATE INDEX CONCURRENTLY IF NOT EXISTS granules_producer_granule_id_index ON granules(producer_granule_id)');
```

Comment:
Move the `CREATE INDEX` statement inside the condition that checks if `granules.producer_granule_id` does not exist, so the stack can be deployed when the producer_granule_id column has been added regardless if the index is added as well.

Reply:
Yes, the intention is that in the production environment, the migration can be run in steps. After the column has been added, the stack can be deployed, and populating the data and indexing the column can happen at a later time.